src/jaegertracing/baggage/RemoteRestrictionJSON.h

```diff
@@ -18,10 +18,17 @@
 #define JAEGERTRACING_BAGGAGE_REMOTERESTRICTIONJSON_H
 
 #include <cstdint>
-#include <nlohmann/json.hpp>
 #include <string>
 #include <vector>
 
+#include "jaegertracing/Constants.h"
+/* nlohmann json changed locations (in 2.1.0?) */
+#ifdef JAEGERTRACING_NLOHMANN_UNQUALIFIED_PATH
+  #include <json.hpp>
+#else
+  #include <nlohmann/json.hpp>
+#endif
```

Comment:
Interesting. Another possibility is including the header somewhere because I think it is a header-only library.

Reply:
Removed from latest push since cpp-client doesn't build against older headers due to code compat issues anyway (#47)